From a09f942074d466f9b0262d6d7f24b45a499794fc Mon Sep 17 00:00:00 2001
From: Nitisha <nitisha.tomar@intel.com>
Date: Thu, 24 May 2018 19:30:41 +0530
Subject: [PATCH 18/21] [SM] make SD card visible for apps Ported from:
 https://android-review.googlesource.com/c/platform/frameworks/base/+/396052

Mount Flags is coming empty due to the visibile problem.

Change-Id: If88b44f3c89af4e8b81482b82bd7ee1c773f2fd9
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-62477
Signed-off-by: Nitisha <nitisha.tomar@intel.com>
---
 services/core/java/com/android/server/StorageManagerService.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/StorageManagerService.java b/services/core/java/com/android/server/StorageManagerService.java
index 1ca0112..bdd18d4 100644
--- a/services/core/java/com/android/server/StorageManagerService.java
+++ b/services/core/java/com/android/server/StorageManagerService.java
@@ -1316,7 +1316,7 @@ class StorageManagerService extends IStorageManager.Stub
 
             // Adoptable public disks are visible to apps, since they meet
             // public API requirement of being in a stable location.
-            if (vol.disk.isAdoptable()) {
+            if (vol.disk.isAdoptable() || vol.disk.isSd()) {
                 vol.mountFlags |= VolumeInfo.MOUNT_FLAG_VISIBLE;
             }
 
-- 
1.9.1

